docroot/sites/all/modules/features/forms/templates/check-your-eligibility.tpl.php

```diff
@@ -1,18 +1,18 @@
-<?php print $salutation;?>
+<?php print $salutation;?><br><br>
 
 Thank you! We have received your completed pre-assessment for the new Cornell University ILR School
 Executive Master of Human Resource Management, and look forward to evaluating your information. We
 will email you within 5 to 7 business days to provide feedback on your eligibility and further details
-about the program. Please do not respond to this email, this is a confirmation message only.
+about the program. Please do not respond to this email, this is a confirmation message only.<br><br>
 
-<? print $form_title; ?>
+<? print $form_title; ?><br><br>
 
-<? print $form_data;?>
+<? print $form_data;?><br><br>
 
-Sincerely,
-Graduate Programs Office
-ILR School at Cornell University
-<? print $admin_email; ?>
+Sincerely,<br>
+Graduate Programs Office<br>
+ILR School at Cornell University<br>
+<? print $admin_email; ?><br><br>
 
---
+--<br>
 This is an automated e-mail from <? print $sitename; ?>.
```

Comment:
That was requested by Grad programs for this EMHRM form.

Regards,
Nathan

From: Aaron Froehlich [mailto:notifications@github.com]
Sent: Friday, January 23, 2015 5:07 PM
To: ilrWebServices/ilr-website
Subject: Re: [ilr-website] Adds br tags to the form tpl.php files (#107)


In docroot/sites/all/modules/features/forms/templates/check-your-eligibility.tpl.php<https://github.com/ilrWebServices/ilr-website/pull/107#discussion_r23484176>:

>

> ---

> +--<br>

>  This is an automated e-mail from <? print $sitename; ?>.

Did you want to remove this "automated e-mail" line?

—
Reply to this email directly or view it on GitHub<https://github.com/ilrWebServices/ilr-website/pull/107/files#r23484176>.